src/diagnostics.rs

```diff
@@ -279,6 +280,7 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
                     CreatedCallId(id) => format!("function call with id {}", id),
                     CreatedAlloc(AllocId(id)) => format!("created allocation with id {}", id),
                     FreedAlloc(AllocId(id)) => format!("freed allocation with id {}", id),
+                    DummyOpInIsolation(op) => format!("produced dummy error for `{}`", op),
```

Comment:
I don't know what you mean by "dummy" error. I suggest to say something more like "`{}` was made to return an error due to isolation" or so (and print a hint mentioning `-Zmiri-disable-isolation`).